src/gui-wizard-gtk/main.c

```diff
@@ -200,7 +200,7 @@ int main(int argc, char **argv)
     g_list_free_full(possible_names, free);
 
     /* if we have only 1 workflow, we can use the events from it as default */
-    if (!expert_mode && g_auto_event_list == NULL && g_hash_table_size(possible_workflows) == 1)
+    if (!expert_mode && !g_auto_event_list && g_hash_table_size(possible_workflows) == 1)
```

Comment:
Can we leave it like it was? Explicitly comparing against `NULL` at least lets you know it’s a pointer without looking at the declaration.

Reply:
Will do.